src/auditlog/models.py

```diff
@@ -229,6 +233,62 @@ def changes_str(self, colon=': ', arrow=smart_text(' \u2192 '), separator='; '):
 
         return separator.join(substrings)
 
+    @property
+    def changes_display_dict(self):
```

Comment:
Can you inline-comment this? Than it is better understandable what it exactly does.